From 85ded95962b70883aecffe6978a909d57d425c1a Mon Sep 17 00:00:00 2001
From: kgopala2 <karthik.l.gopalakrishnan@intel.com>
Date: Mon, 25 Feb 2019 22:07:53 +0800
Subject: [PATCH 1298/1417] media: i2c: max9286: form max9286 node names based
 on pdata suffix

Rx port index ID was being used to name MAX9286 instances. Now
modified it to use suffix variable from platform-date.

Change-Id: I8cea76a18883b416743b4a782596db6986d69560
Signed-off-by: kgopala2 <karthik.l.gopalakrishnan@intel.com>
---
 drivers/media/i2c/ici/max9286_ici.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/media/i2c/ici/max9286_ici.c b/drivers/media/i2c/ici/max9286_ici.c
index 4345c70..cc82892 100644
--- a/drivers/media/i2c/ici/max9286_ici.c
+++ b/drivers/media/i2c/ici/max9286_ici.c
@@ -497,7 +497,7 @@ static int init_ext_sd(struct i2c_client *client, struct max9286_subdev *max_sd,
 	}
 	maxim = to_max_9286(subdev);
 
-	snprintf(name, sizeof(name), "MAX9286 %d", idx);
+	snprintf(name, sizeof(name), "MAX9286 %c", maxim->pdata->suffix);
 
 	strncpy(maxim->sub_devs[idx].sd_name, name, sizeof(name));
 
@@ -699,7 +699,7 @@ static int max9286_register_subdev(struct max9286 *max, struct i2c_client *clien
 		max->ffmts[i]->pixelformat = ICI_FORMAT_SGRBG10;
 		max->ffmts[i]->field = ICI_FIELD_NONE;
 		snprintf(max->sub_devs[i].sd_name, sizeof(max->sub_devs[i].sd_name),
-			"MAX9286 %d", i);
+			"MAX9286 %c", max->pdata->suffix);
 	}
 	return 0;
 }
-- 
2.7.4

